ui/src/lib/actions.ts

```diff
@@ -369,3 +371,34 @@ export const sendStackTrace = async (stackTrace: string): Promise<EmailResult> =
     const endpoint = `${baseUrl}/email/send/stack-trace`;
     return postRequest<EmailResult>(endpoint, currentUser.uid, stackTrace, 'text/plain');
 };
+
+/**
+ * Get all the members of an owned grouping through paginated calls.
+ *
+ * @param groupPaths - The paths to the groups
+ * @param page - Page of the group (optional)
+ * @param size - Size of the group (optional)
+ * @param sortString - String to sort by column name
+ * @param isAscending - On true the data returns in ascending order
+ *
+ * @returns The promise of members of an owned grouping
+ */
+export const getGroupingMembers = async (
```

Comment:
Since this is in the `actions.ts` file, this function should have it's params validated. This is because server actions are exposed endpoints that can be a security issue. Like the other functions in this file, use zod to validate the params, thanks.